2-alcance/README.md

```diff
@@ -1,3 +1,187 @@
+
+Paula encuentra un sitio web que muestra la realidad de la sociedad ante la igualdad de género, encuentra entrevistas con expertos (psicólogos, abogados) que tratan temas de violencia de género y sobre todo encuentra historias reales de otras mujeres que le hacen recapacitar que ella misma está viviendo una relación insana y que no puede permitirlo. Debido al impacto que tuvo para Paula este sitio web, decide ponerse en contacto con *hoyportimañanapormi.com* para pedir más información y para trasladarles su historia para que la publicasen con el objetivo de ayudar a otras personas como lo hicieron las historias de otras mujeres a ella. Por lo tanto, Paula es un usuario voluntario ya que se ha registrado en el sistema, cuenta con su propia cuenta y puede compartir contenido en la página web.
+
+#### 2.2. Escenario 2: Juan y su trabajo para la universidad
+Juan tiene 21 años y es un estudiante de criminología. Se encuentra en cuarto curso y en unos meses termina sus estudios universitarios. Para la universidad debe realizar un trabajo sobre diferentes casos de violencia de género y para ello decide acceder a la web hoyportimañanapormi.com. Es una página web muy completa, ya que dispone de diferentes apartados y todos muy útiles para que pueda elaborar su trabajo. En la web puede encontrar: noticias, artículos, reportajes, entrevistas, historias reales y consejos. Es una web que ya conocía de hace tiempo ya que es un tema que le interesa y le preocupa.
+
+La web le resulta muy útil (ya que su contenido es actualizado constantemente), tanto para el trabajo universitario como para enriquecerse personalmente. Pedro es un usuario no registrado que simplemente accediendo a la página web podrá encontrar todo el contenido que deseé sin tener que registrarse. El incoveniente de no estar registrado es que no podrá actualizar el contenido
+
+#### 2.3. Escenario 3: Dolores y sus ganas de aprender
+Dolores tiene 65 años y mientras horneaba un bizcoho para celebrar el cumpleaños de su amiga Encarna ha visto en las noticias un nuevo caso de desigualdad de género por parte de la empresa Inditex. A Dolores le encanta mantenerse informada y con sus escasos conocimientos de informática ha decidido indagar en su ordenador de mesa que poseé. Tecleando e investigando un poco a encontrado la página web perfecta para seguir informándose: hoyportimañanapormi.com. No es solo una página web que se mantiene en constante actualización, es facil de usar y muy legible justo lo que Dolores necesita. Dolores se ha quedado encantada con la página web y le ha parecido tan facil de usar que no ha dudado ni un segundo en registrarse para poder añadir contenido y empezar a colaborar ¡qué animada!. Por lo tanto, ha pasado de ser usuario no registrado a usuario voluntario. Dolores ya tiene una nueva página web favorita hoyportimañanapormi.com y como no, lasrecetasdemj.com.
+
+
+### 3. NO OBJETIVOS
+Esta versión *no* incluirá las siguientes funciones:
```

Comment:
Aquí deberían incluirse solamente cuestiones que sería razonable plantearse para este sitio. En ningún momento parece que planteéis un sitio de venta, así que no haría falta incluir "Compra de productos" (lo mismo que no haría falta concretar que no haréis cosas como "Instalar un condensador de fluzo en el Delorean").